### PR TITLE
change education select option

### PR DESCRIPTION
**Pull request by hughes-dustin**

### CODE REVIEW

File: src/schemas/22-0994/schema.js [hunk 1]
```diff
@@ -1,10 +1,7 @@
 import _ from "lodash/fp";
 import { countries } from "../../common/constants";
 import definitions from "../../common/definitions";
-import {
-  pciuCountries,
-  pciuStates
-} from "../../common/constants";
+import { pciuCountries, pciuStates } from "../../common/constants";
```
Comment (zurbergram) [inline src/schemas/22-0994/schema.js]: looks like your local linter changed a bunch of things

File: src/schemas/22-0994/schema.js [hunk 2]
```diff
@@ -29,8 +26,8 @@ const schema = {
       ...definitions.address,
       properties: {
         ...definitions.address.properties,
-        street3: definitions.address.properties.street2,
-      },
+        street3: definitions.address.properties.street2
+      }
```
Comment (zurbergram) [inline src/schemas/22-0994/schema.js]: looks like your local linter changed a bunch of things

File: src/schemas/22-0994/schema.js [hunk 3]
```diff
@@ -69,44 +66,44 @@ const schema = {
...
     hasSelectedPrograms: {
-      type: "boolean",
+      type: "boolean"
     },
     bankAccount: {
       $ref: "#/definitions/bankAccount"
     },
     prefillBankAccount: {
-      type: 'object',
+      type: "object",
```
Comment (zurbergram) [inline src/schemas/22-0994/schema.js]: looks like your local linter changed a bunch of things

**Comment by hughes-dustin** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

**Pull request merged**